semeio/jobs/overburden_timeshift/ots.py

```diff
@@ -270,9 +272,9 @@ def add_survey(self, name, date):
 
     @staticmethod
     def _divide_negative_shift(ts, div_val=5.0):
-        for i in range(len(ts)):
-            if ts[i] < 0:
-                ts[i] /= div_val
+        for index, _ in enumerate(ts):
+            if ts[index] < 0:
+                ts[index] /= div_val
```

Comment:
The last line of proposed fix is debatable
```suggestion
        for index, value in enumerate(ts):
            if value < 0:
                ts[index] = value/div_val
```

Reply:
good